deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -841,80 +476,437 @@ collectors:
+              action: delete
+            - key: process.executable.path
+              action: delete
+        batch:
+          send_batch_size: 1000
+          timeout: 1s
+          send_batch_max_size: 1500
+        batch/aggs:
+          send_batch_size: 16384 # 2x the default
+          timeout: 10s
+        # [Elastic Trace Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/elastictraceprocessor)
+        elastictrace: {} # The processor enriches traces with elastic specific requirements.
+        # [LSM Interval Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/lsmintervalprocessor)
+        lsminterval:
+          intervals:
+            - duration: 1m
+              statements:
+                - set(resource.attributes["metricset.interval"], "1m")
+                - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "1m"], "."))
+                - set(attributes["processor.event"], "metric")
+            - duration: 10m
+              statements:
+                - set(resource.attributes["metricset.interval"], "10m")
+                - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "10m"], "."))
+                - set(attributes["processor.event"], "metric")
+            - duration: 60m
+              statements:
+                - set(resource.attributes["metricset.interval"], "60m")
+                - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "60m"], "."))
+                - set(attributes["processor.event"], "metric")
+      exporters:
+        debug: {}
+        # [Elasticsearch exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/elasticsearchexporter/README.md)
+        elasticsearch/otel:
+          endpoints: # List of Elasticsearch endpoints.
+            - ${env:ELASTIC_ENDPOINT}
+          api_key: ${env:ELASTIC_API_KEY} # API key for Elasticsearch authentication.
+          logs_dynamic_index:
+            enabled: true
+          # Enable in order to skip the SSL certificate Check
+          # tls:
+          #   insecure_skip_verify: true
+          mapping:
+            mode: otel
+        elasticsearch/ecs:
+          endpoints:
+            - ${env:ELASTIC_ENDPOINT}
+          api_key: ${env:ELASTIC_API_KEY}
+          mapping:
+            mode: ecs
+      service:
+        pipelines:
+          metrics:
+            receivers: [otlp]
+            exporters: [routing]
+          metrics/infra/ecs:
+            receivers: [routing]
+            processors:
+              - elasticinframetrics
+              - attributes/dataset
+              - resource/process
+              - batch
+            exporters: [debug, elasticsearch/ecs]
+          metrics/otel:
+            receivers: [routing]
+            processors: [batch]
+            exporters: [debug, signaltometrics, elasticsearch/otel]
```

Comment:
I'm a bit on the fence whether to send metrics through the signaltometrics connector as we don't have a great experience for metrics-only services. I think I'd lean towards removing it now as we can always easily add it back in later. But either way, I don't see this as blocking.

Reply:
Let's remove then for now https://github.com/elastic/elastic-agent/pull/6444/commits/c83cd8f1550101dea5bcd9ac5885a495e4c49561

Thanks for the review!